src/scirpy/tl/_mutational_load.py

```diff
@@ -0,0 +1,181 @@
+
+    if frequency:
+        frequency_string = "mu_freq"
+    else:
+        frequency_string = "mu_count"
+    if ignore_chars == "None":
+        ignore_chars = [".", "N"]
+
+    if region == "IMGT_V(D)J":
+        for chain in chains:
+            mutations = []
+            for row in range(len(airr_df)):
+                mutation = simple_hamming_distance(
+                    airr_df.iloc[row].loc[f"{chain}_{sequence_alignment}"],
+                    airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"],
+                    frequency=frequency,
+                    ignore_chars=ignore_chars,
+                )
+                mutations.append(mutation)
+
+            if inplace:
+                params.set_obs(f"{chain}_IMGT_V(D)J_{frequency_string}", mutations)
+
+            else:
+                mutation_df[f"{chain}_IMGT_V(D)J_{frequency_string}"] = mutations
+                return mutation_df
+
+    # calculate SHM up to nucleotide 312. Referring to the IMGT unique numbering scheme this includes:
+    # fwr1, cdr1, fwr2, cdr2, fwr3, but not cdr3 and fwr4
+    if region == "IMGT_V_segment":
+        for chain in chains:
+            mutations = []
+            for row in range(len(airr_df)):
+                v_region_germline = airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"][:312]
+                v_region_sequence = airr_df.iloc[row].loc[f"{chain}_{sequence_alignment}"][:312]
+
+                mutation = simple_hamming_distance(
+                    v_region_sequence, v_region_germline, frequency=frequency, ignore_chars=ignore_chars
+                )
+                mutations.append(mutation)
+
+            if inplace:
+                params.set_obs(f"{chain}_v_segment_{frequency_string}", mutations)
+
+            else:
+                mutation_df[f"{chain}_v_segment_{frequency_string}"] = mutations
+                return mutation_df
+
+    if region == "subregion":
+        subregion_df = get_airr(params, ["fwr1", "fwr2", "fwr3", "fwr4", "cdr1", "cdr2", "cdr3"], chains)
+
+        for chain in chains:
+            airr_df[f"{chain}_junction_len"] = [len(a) for a in airr_df[f"{chain}_junction"]]
+
+            mutation_dict = {"fwr1": [], "fwr2": [], "fwr3": [], "fwr4": [], "cdr1": [], "cdr2": [], "cdr3": []}
+
+            for row in range(len(airr_df)):
+                regions = {
+                    "fwr1": (0, 78),
+                    "cdr1": (78, 114),
+                    "fwr2": (114, 165),
+                    "cdr2": (165, 195),
+                    "fwr3": (195, 312),
+                    "cdr3": (312, 312 + airr_df.iloc[row].loc[f"{chain}_junction_len"] - 6),
+                    "fwr4": (
+                        312 + airr_df.iloc[row].loc[f"{chain}_junction_len"] - 6,
+                        len(airr_df.iloc[row].loc[f"{chain}_{germline_alignment}"]),
+                    ),
+                }
+
+                for v, coordinates in regions.items():
```

Comment:
```suggestion
                for region, coordinates in regions.items():
```

One letter loop variables should only be used if they follow certain conventions, e.g. `i/j/k` for counters in for loops, 
or `k, v` for key, value pairs from `dict.items()`. 

Since you use `v` for the dict key, this can be confusing and I suggest to use a "proper" variable name like region here.